### DISCUSSION TITLE
Display full screen mode by default

### DISCUSSION THREAD

**Discussion by lihong-xmuseum** [mozilla/hubs#4270] [Hubs Cloud Questions] [answered]
Customize how to set the default full-screen mode for the client. Modify some content and find that it does not work.

**Comment by lihong-xmuseum** [comment, marked as answer]
resolved

**Reply by Softvision-GeluHaiduc** [reply]
Marking this as answered, if you feel comfortable you can share some details here on how you resolved the issues you encountered. Thank you!
